syft/source/location.go

```diff
@@ -11,7 +11,7 @@ import (
 // Location represents a path relative to a particular filesystem resolved to a specific file.Reference. This struct is used as a key
 // in content fetching to uniquely identify a file relative to a request (the VirtualPath).
 type Location struct {
-	Coordinates
+	Coordinates `cyclonedx:""`
```

Comment:
Empty string here causes the sub-properties to be included, without an intermediate property name segment, like `syft:location:0:coordinates:path`

Reply:
this should get captured as a comment on this field in the code for certain